src/impls/from_iterator.rs

```diff
@@ -19,6 +19,17 @@ where
     {
         Self::from_iter(iterable)
     }
+
+    /// Create a new `Counter` initialized with the given iterable.
+    /// Allocate with iterator size hints, defaulting to the lowerbound
+    pub fn init_with_capacity<I>(iterable: I, capacity: usize) -> Self
```

Comment:
Sorry, thought I was clear earlier. Given that `init` is now deprecated, we can simply omit `init_with_capacity`.

Reply:
done.